Dockerfile

```diff
@@ -13,15 +13,22 @@ RUN go mod download
 COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
+COPY cmd/ cmd/
+COPY pkg/ pkg/
 
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o vgmanager cmd/vgmanager/main.go
 
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+# vgmanager needs 'nsenter' and other basic linux utils to correctly function
+FROM centos:8
 WORKDIR /
 COPY --from=builder /workspace/manager .
+COPY --from=builder /workspace/vgmanager .
 USER 65532:65532
 
-ENTRYPOINT ["/manager"]
+# '/manager' is lvm-operator entrypoint
+# ENTRYPOINT ["/manager"]
+
+# '/vgmanager' is vgmanager entrypoint which is used in daemonset image
+# ENTRYPOINT ["/vgmanager"]
```

Comment:
I think you can only have a single entrypoint per container.

Reply:
> I think you can only have a single entrypoint per container image

Reply:
- we are invoking different binaries in `command` field of pod spec.

Reply:
The entrypoint is the default command that is invoked so there should be only one. I think that will still allow us to use the vgmanager command in a different container explicitly.

Reply:
Yes, I thought explicit is better than implicit and commented out both entries, they are there just for ref

Reply:
As per https://docs.docker.com/engine/reference/builder/ we will need an entrypoint. Please uncomment the one for the manager